### DISCUSSION TITLE
Support for isotope-labeled data?

### DISCUSSION THREAD

**Discussion by roberthardt** [fgcz/prolfqua#60] [Q&A]
Dear developers,

can I also analyze DIA or DDA-data run through MaxQuant or Spectronaut, which has isotope-labeled channel intensity data? I would guess this should work, because in the end of the day it shouldn't matter that much, because I have intensities for each channel in a separate column (wide data frame) or a column indicating the channel (long data).
One might have to adapt change column names, but that should be easily doable.

Best 

Robert

**Comment by wolski** [comment]
Hi Robert, 

Yes, but it depends on your use of the heavy channel. Are you using it for normalization (heavy peptides SRM style), or is the light channel one condition and the heavy channel the second condition?
Can you explain the design in more detail?

If you have the heavy samples for normalization only, I would run two analyses with the light only and then with the normalized light.
For SRM normalization you might look at this vignette:
https://fgcz.github.io/prolfqua/articles/Supplement_TidyAnalysis_HL_Skyline.html


However, If the light is one condition and the heavy the other, and you have several measurements (bio reps) then you have to convert it into a long format and use the heavy light as one explanatory variable, and the sample as the second and run a paired analysis.

Best regards
Witold

**Reply by roberthardt** [reply]
Hi Witold,

i intend to use it for the latter case. Some colleagues of mine here at the institute regularly do IP experiments with control vs bait, where one condition is labeled light and the other heavy using SILAC. They usually also have replicates. So according to your information the primary variable is the label status and the secondary the sample/file. Perfect. This should be easy to implement. 

Best and thanks for the prompt reply

Robert
